reverb/cc/platform/default/repo.bzl

```diff
@@ -219,17 +228,20 @@ filegroup(
 def _tensorflow_solib_repo_impl(repo_ctx):
     tf_lib_path = _find_tf_lib_path(repo_ctx)
     repo_ctx.symlink(tf_lib_path, "tensorflow_solib")
+    suffix = "so.2"
+    if is_darwin(repo_ctx):
+        suffix = "2.dylib"
+
     repo_ctx.file(
         "BUILD",
         content = """
 cc_library(
     name = "framework_lib",
-    srcs = ["tensorflow_solib/libtensorflow_framework.so.2"],
+    srcs = ["tensorflow_solib/libtensorflow_framework.{}"],
```

Comment:
does need to be filled with suffix somehow?

Reply:
oh i see it below.  can you use named templates in this code?  i.e.,  `...{suffix}....format(suffix=suffix)`?